src/main/resources/sql/schema.pgsql

```diff
@@ -38,7 +38,8 @@ CREATE TABLE IF NOT EXISTS orders(
     ID SERIAL PRIMARY KEY NOT NULL,
     CREATED_DATE DATE NOT NULL,
     DUE_DATE DATE NOT NULL,
-    DELIVERY_LOCATION VARCHAR NOT NULL
+    DELIVERY_LOCATION VARCHAR NOT NULL,
+    ORDER_TYPE VARCHAR NOT NULL
 );
 
 CREATE TABLE IF NOT EXISTS BILL_OF_MATERIAL(
```

Comment:
This table is already on main, no need to add it.

You just to pull from main and resolve merge conflicts to incorporate the latest changes into your branch